docs/tutorials/avro.ipynb

```diff
@@ -544,7 +544,7 @@
         "id": "IF_kYz_o2DH4"
       },
       "source": [
-        "A comprehensive set of examples of Avro dataset API is provided within <a target=\"_blank\" href=\"https://github.com/tensorflow/io/blob/master/tests/test_parse_avro_eager.py#L580r\">the tests.</a>\n"
+        "A comprehensive set of examples of Avro dataset API is provided within <a target=\"_blank\" href=\"https://github.com/tensorflow/io/blob/master/tests/test_parse_avro.py\">the tests</a>.\n"
```

Comment:
change it to `https://github.com/tensorflow/io/blob/master/tests/test_parse_avro.py#L437`